_layouts/responsive.html

```diff
@@ -19,18 +19,12 @@
 </head>
 
 <body>
-    <div class="banner" style="
-            text-align: center;
-            background: #fff;
-            font-size: 19px;
-            line-height: 28px;
-            padding: 10px;
-            border-top: solid 2px #aaa;
-            border-bottom: solid 2px #aaa;
-            margin-top: 15px;
-            color: #222;">
-                <span style="color:#000;">Tickets now open! &nbsp; <a href="/purchase/" class="btn btn-primary">Buy Tickets</a></span>
+   {% if site.tickets_for_sale or site.waiting_list_enable %}
+    <div class="banner" style="text-align: center; background: #fff; font-size: 19px; line-height: 28px; padding: 10px; border-top: solid 2px #aaa; border-bottom: solid 2px #aaa; margin-top: 15px; color: #222;">
+        {% if site.tickets_for_sale %}<span style="color:#000;">Tickets now open! &nbsp; <a href="/purchase/" class="btn btn-primary">Buy Tickets</a></span>{% endif %}
+        {% if site.waiting_list_enable %}<span style="color:#000;">Tickets are sold out! &nbsp; <a href="{{ site.waiting_list_url }}" class="btn btn-primary">Signup for the waiting list</a></span>{% endif %}
```

Comment:
Seems like these two variables are diametric and could be a single boolean.  Or, what you more likely want:

```
if tickets_for_sale:
    if not waiting_list_enable
        Tickets now open!
    else
        Tickets are sold out!
    endif
endif